src/dialogs.ts

```diff
@@ -230,11 +230,9 @@ export function choose(items: string[], options: IChooseOptions|string = {}): Pr
   });
 }
 
-export interface IAreYouSureOptions {
-  title?: string;
+export interface IAreYouSureOptions extends Pick<IDialogOptions, 'title' | 'additionalCSSClasses'> {
```

Comment:
What does `Pick<IDialogOptions, 'title' | 'additionalCSSClasses'>`, or to be more specific `'title' | 'additionalCSSClasses'` mean? Why do you need to add this?

Reply:
It creates a new interface with the properties `title` and `additionalCSSClasses` from the `IDialogOptions`, which is then used to extend the current `IAreYouSureOptions`. Therefore, these two props are now always tied to `IDialogOptions` (i.e. always in sync). I guess we don't want to extend it completely because `placeholder` would be unused, and `primaryBtnText` is actually called `button` in the existing interface?

Reply:
Right. @lehnerchristian could you please address @dvmichaelpuehringer request?

Reply:
as michael has elaborated as I understood it `Pick` creates a new interface out of `title` and `additionalCSSClasses`, so the resulting interface  should be

```
interface IAreYouSureOptions {
  button?: string;
  cancelButton?: string;
  title?: string;
  additionalCSSClasses?: string;
}
```

what do you mean with `are now always tied to `IDialogOptions`, @dvmichaelpuehringer ? that they always have the type from `IDialogOptions`?

Reply:
Yes, which is exactly what we want I suppose.

Reply:
yes, I think so. @thinkh, then I think you can merge if everything is ok with this PR

Reply:
Sure.